certora/mutate/README.md

```diff
@@ -0,0 +1,52 @@
+# Using Certora Mutate
```

Comment:
it seems that the whole `./certora/mutate/` directory needs updating. do we want to merge it as is now and update later?

Reply:
I think it may be best to just delete this directory and start fresh once we are ready for mutation testing (if we would like to do this). We will keep our old repo so I can always look back at the manual mutants we wrote last time if we want to reintroduce them. I will speculatively delete this directory now.